conf/version_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2024 Northern.tech AS
+//
+//	Licensed under the Apache License, Version 2.0 (the "License");
+//	you may not use this file except in compliance with the License.
+//	You may obtain a copy of the License at
+//
+//	    http://www.apache.org/licenses/LICENSE-2.0
+//
+//	Unless required by applicable law or agreed to in writing, software
+//	distributed under the License is distributed on an "AS IS" BASIS,
+//	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//	See the License for the specific language governing permissions and
+//	limitations under the License.
+package conf
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMenderShellVersionUnknown(t *testing.T) {
+	Version = ""
+	v := VersionString()
+	assert.Equal(t, v, "unknown")
+}
+
+func TestMenderShellVersion(t *testing.T) {
+	Version = "1.0"
+	v := VersionString()
+	assert.Equal(t, v, Version)
+}
+
+func TestMenderShowVersion(t *testing.T) {
+	Version = "v1.0"
+	v := ShowVersion()
+	assert.True(t, strings.Contains(v, "v1.0"))
+}
```

Comment:
These tests have very little value: we are setting a variable and then checking that it is returned.

Can we instead move it into the `cli_test.go` package and check that `mender-snapshot --version` prints the right thing? You can look at [mender-artifact cli tests](https://github.com/mendersoftware/mender-artifact/blob/master/cli/cli_test.go) for some inspiration